src/components/button/_button.scss

```diff
@@ -179,8 +179,6 @@
 @mixin button--x {
   // <button> elements cannot be used as flex containers
   button.#{$prefix}--btn {
-    display: inline-block;
```

Comment:
Is this removal intentional...?

Reply:
I believe the default display value for button elements is `inline-block` right?

Reply:
You are right wrt the user-agent stylesheet thing, but I'd guess it's for protecting against application-level overrides. Tracing the code history tells me that the addition was intentional, and I wouldn't remove it unless I have a stronger reason.

Reply:
sounds good, I added that rule back